config/schedule.rb

```diff
@@ -26,3 +26,10 @@
   set :output, standard: nil, error: 'log/cv-err.log'
   rake "cv:all_roots"
 end
+
+every :hour, roles: [:cache_cleaner] do
+  set :output, standard: 'log/zip_cache_cleanup.log'
+  command <<-'END_OF_COMMAND'
+    find /sdr-transfers -mindepth 5 -type f -name "*.zip" -mtime +1 -exec bash -c 'TARGET="{}"; rm -v ${TARGET%ip}*' \;
```

Comment:
I'd like to suggest setting `/sdr-transfers` via the `zip_command` setting. We did some similar work in the `preservation_robots` (https://github.com/sul-dlss/preservation_robots/blob/master/config/schedule.rb#L1-L2) to hook scheduled tasks up to settings.

Reply:
Sorry, not `zip_command`, but `zip_storage`. I edited the above comment accordingly.